private/bufpkg/bufcheck/buflint/testdata/protovalidate/proto/cel_field.proto

```diff
@@ -8,140 +8,155 @@ import "google/protobuf/timestamp.proto";
 
 message Foo {
   required double compile_fail = 1 [(buf.validate.field).cel = {
-    id: "Foo.compile_fail",
-    message: "compile_fail_message",
+    id: "Foo.compile_fail"
+    message: "compile_fail_message"
```

Comment:
I think `buf format` has been updated? Or my auto-format wasn't enabled this whole time. The former is more likely.

Reply:
Also happy to revert these changes but I think they look nicer without the commas.